components/automate-cli/pkg/verifyserver/server/api/v1/startMockServer.go

```diff
@@ -0,0 +1,52 @@
+package v1
+
+import (
+	"fmt"
+
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/services/startmockserverservice"
+	"github.com/gofiber/fiber"
+)
+
+func (h *Handler) StartMockServer(c *fiber.Ctx) {
+
+	reqBody := new(models.StartMockServerRequestBody)
+	err := c.BodyParser(&reqBody)
+
+	// If request body is invalid
+	if err != nil {
+		errString := fmt.Sprintf("start mock-server request body parsing failed: %v", err.Error())
+		h.Logger.Error(fmt.Errorf(errString))
+		c.Status(fiber.StatusBadRequest).JSON(fiber.Map{"message": "Invalid request body"})
```

Comment:
Please use like this everywhere you are returning error `c.Next(&fiber.Error{Code: http.StatusBadRequest, Message: "Invalid request body"})`